openedx/core/djangoapps/notifications/views.py

```diff
@@ -291,13 +292,18 @@ def get(self, request):
         count_total = 0
         count_by_app_name_dict = {}
         show_notifications_tray_enabled = False
+        notification_apps = get_notifications_app_names_list(COURSE_NOTIFICATION_APPS)
 
         for item in count_by_app_name:
             app_name = item['app_name']
             count = item['count']
             count_total += count
             count_by_app_name_dict[app_name] = count
 
+        for app_name in notification_apps:
+            if app_name not in count_by_app_name_dict:
+                count_by_app_name_dict[app_name] = 0
+
```

Comment:
I think the better way would be to initialize count_by_app_name_dict with all apps count as 0

```
count_by_app_name_dict = {
    app_name: 0
    for app_name in COURSE_NOTIFICATION_APPS.keys()
}
```
Then update the count of all the apps which have count greater than 0
```
for item in count_by_app_name:
    app_name = item['app_name']
    count = item['count']
    count_total += count
    count_by_app_name_dict[app_name] = count
```

Reply:
@muhammadadeeltajamul why would this be better?